config/openhpc.yml

```diff
@@ -101,3 +101,21 @@ openhpc_packages:
   - infiniband-diags
   - emacs
 #  - git-lfs
+
+users:
```

Comment:
Can we introduce a layer of indirection here (eg, cluster_users or cluster_admins, whatever) in case we have different kinds of users - or should we simply make the distinction between the different kinds of users through the groups of which they are members?